#include <visual_servoing_with_safe_interaction/RPY.h>

int32 id
geometry_msgs/Vector3 position
RPY orientation
